simplq/src/components/pages/Home/LandingPage.jsx

```diff
@@ -18,8 +18,8 @@ export default () => {
         <Header className={styles['main-header']}>SimplQ</Header>
         <p className={styles.subtitle}>{subtitle}</p>
         <p className={styles.description}>
-          Create and manage queues with our free online queue management solution for easy and
-          instant crowd control
+          Web based free queue management software to let your customers feel more relaxed and
+          delighted.
         </p>
```

Comment:
Let's just say Web based free queue management software?

Reply:
You mean to drop the "to let your customers feel more relaxed and delighted."? I was liking it.